docs/databags.md

```diff
@@ -109,6 +109,7 @@ An apps hash is a contextual (see above) Hash, part of a plaintext data bag item
 * `[app]['download']['version']` - Expected [version number][app.conf] (if any) used to determine if a new app should be downloaded.
 * `[app]['files']` - Hash of files to manage under the "default" or "local" directory.
 * `[app]['files'][filename]` - Contents of a particular file to manage. It can take 3 values, a hash of stanzas -> key-value pairs (then written with the splunk template), a string (written as is), or nil / false (deleted). If the hash or string is empty, the file is also deleted.
+* `[app]['lookups']` - Hash of lookup files for the app. The key in the hash will be the name of file when it lands on splunk and value will be the url to lookup file. To delete an existing lookup file, set the value of the lookup file to empty or to `"nil"`. Only supported file name formats are .csv, .gz and .kmz. If you are downloading a comma separated text file (.txt), the file name you specify as the key in the hash should have a .csv extension.  
```

Comment:
> Only supported file name formats are .csv, .gz and .kmz. If you are downloading a comma separated text file (.txt), the file name you specify as the key in the hash should have a .csv extension.

Where is this requirement from? How is it being enforced?

> To delete an existing lookup file, set the value of the lookup file to empty or to `"nil"`.

To the string nil ? and not just boolean false or null?